boards/saml21-xpro/include/gpio_params.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2020 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup   boards_saml21-xpro
+ * @{
+ *
+ * @file
+ * @brief     Board specific configuration of direct mapped GPIOs
+ *
+ * @author    Benjamin Valentin <benjamin.valentin@ml-pa.com>
+ */
+
+#ifndef GPIO_PARAMS_H
+#define GPIO_PARAMS_H
+
+#include "board.h"
+#include "saul/periph.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief    GPIO pin configuration
+ */
+static const  saul_gpio_params_t saul_gpio_params[] =
+{
+    {
+        .name = "LED(orange)",
+        .pin = LED0_PIN,
+        .mode = GPIO_OUT,
+        .flags = SAUL_GPIO_INVERTED
```

Comment:
Are you sure you don't need  `SAUL_GPIO_INIT_CLEAR` as well for both the LED and the button ? This is done like this on samr30 and samr34 xpro boards (also l21 based).

```suggestion
        .flags = SAUL_GPIO_INVERTED | SAUL_GPIO_INIT_CLEAR
```

Reply:
`board.c` already does `LED0_OFF` in `board_init()`

Reply:
Hmm, ok